filecheck/checker.ts

```diff
@@ -46,12 +49,9 @@ export async function checkFile(filePath, options: CheckerOptions = {}) {
   if (!stat.size) {
     throw new Error(`${filePath} is 0 bytes`);
   }
-  if (stat.size > MAX_FILE_SIZE) {
-    const formatted =
-      stat.size > 1024 * 1024
-        ? `${(stat.size / 1024.0 / 1024).toFixed(1)}MB`
-        : `${(stat.size / 1024.0).toFixed(1)}KB`;
-    const formattedMax = `${(MAX_FILE_SIZE / 1024.0 / 1024).toFixed(1)}MB`;
+  const formattedMax = `${(MAX_FILE_SIZE / 1024.0 / 1024).toFixed(1)}MB`;
```

Comment:
Probably makes sense to reuse `formatSize()` here:

```suggestion
  const formattedMax = formatSize(MAX_FILE_SIZE);
```

Reply:
This would be about `488.3KB`, for we have set `MAX_FILE_SIZE` as 500*1000:

https://github.com/mdn/yari/blob/d2e271092535e9d952e15b0dd76ae8935c938ccc/libs/env/index.js#L119

should we update this to `500*1024`?

Reply:
Good catch, and yeah, let's do that. :)